include/knowhere/route.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2019-2023 Zilliz. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under the License.
+
+#pragma once
+
+#include <string>
+
+#include "comp/index_param.h"
+#include "version.h"
+
+namespace knowhere {
+bool UseDiskLoad(std::string /*index_type*/, std::string /*version*/);
```

Comment:
Can I ask where to use this?

Reply:
Milvus use this to determine load from disk or memory buffer.